internal/cli/utils.go

```diff
@@ -86,6 +86,62 @@ func parsePageRange(pageRange string) ([]int, error) {
 	return pages, nil
 }
 
+func parsePageRangeUnsorted(pageRange string) ([]int, error) {
+	var pages []int
+
+	rngs := strings.Split(removeSpaces(pageRange), ",")
+	for _, rng := range rngs {
+		if rng == "" {
+			continue
+		}
+
+		indices := strings.Split(rng, "-")
+
+		lenIndices := len(indices)
+		if lenIndices > 2 {
+			return nil, errors.New("invalid page range")
+		}
+		if lenIndices == 2 {
+			start, err := strconv.Atoi(indices[0])
+			if err != nil {
+				return nil, errors.New("invalid page number")
```

Comment:
it might be better to rephrase this as
```suggestion
				return nil, errors.New("invalid start page number")
```